src/content/docs/http-api/resources/web-renderer.mdx

```diff
@@ -1,3 +1,63 @@
 ---
```

Comment:
consistent file naming (fix it in sdk to make sure not to break any links when renaming)